packages/engine/crawlers/tezos/index.ts

```diff
@@ -28,7 +31,7 @@ export const tezosEngineRunner: TezosRunner = {
 			item.account.quotes = quotes[makeHashId(item)].quotes;
 		}
 
-		tokenActions.setItems(tokenDocs);
+		setTokens(tokenDocs);
```

Comment:
I am confuse with this `setTokens`, does it update state or storage. If about storage, prefer to putting it at storage package

Reply:
`setTokens` working with storage not state